Doc/library/math.rst

```diff
@@ -26,6 +26,92 @@ The following functions are provided by this module.  Except when explicitly
+:func:`prod`             Product of all the elements.
+:func:`remainder`        Remainder of *x* with respect to *y*.
+:func:`sumprod`          Sum of products from two iterables.
+:func:`trunc`            Integer part.
+:func:`ulp`              Value of the least significant bit.
+
+**Power and logarithmic functions**
+----------------------------------------------------------------------------------------
+:func:`cbrt`             Cube root.
+:func:`exp`              *e* raised to the power *x*.
+:func:`exp2`             *2* raised to the power *x*.
+:func:`expm1`            *e* raised to the power *x*, minus 1.
+:func:`log`              Logarithm of *x* to the given base (*e* by default).
+:func:`log1p`            Natural logarithm of *1+x* (base *e*).
+:func:`log2`             Base-2 logarithm.
+:func:`log10`            Base-10 logarithm.
+:func:`pow`              ``x`` raised to the power ``y``.
+:func:`sqrt`             Square root.
+
+**Trigonometric functions**
+----------------------------------------------------------------------------------------
+:func:`acos`             Arc cosine.
+:func:`asin`             Arc sine.
+:func:`atan`             Arc tangent.
+:func:`atan2`            ``atan(y / x)``.
+:func:`cos`              Cosine.
+:func:`dist`             Euclidean distance.
+:func:`hypot`            Euclidean norm.
+:func:`sin`              Sine.
+:func:`tan`              Tangent.
+
+**Angular conversion**
+----------------------------------------------------------------------------------------
+:func:`degrees`          Convert angle from radians to degrees.
+:func:`radians`          Convert angle from degrees to radians.
+
+**Hyperbolic functions**
+----------------------------------------------------------------------------------------
+:func:`acosh`            Inverse hyperbolic cosine.
+:func:`asinh`            Inverse hyperbolic sine.
+:func:`atanh`            Inverse hyperbolic tangent.
+:func:`cosh`             Hyperbolic cosine.
+:func:`sinh`             Hyperbolic sine.
+:func:`tanh`             Hyperbolic tangent.
+
+**Special functions**
+----------------------------------------------------------------------------------------
+:func:`erf`              `Error function <https://en.wikipedia.org/wiki/Error_function>`_.
+:func:`erfc`             `Complementary error function <https://en.wikipedia.org/wiki/Error_function>`_.
+:func:`gamma`            `Gamma function <https://en.wikipedia.org/wiki/Gamma_function>`_.
+:func:`lgamma`           Natural logarithm of the absolute value of the `Gamma function <https://en.wikipedia.org/wiki/Gamma_function>`_.
+
+**Constants**
+----------------------------------------------------------------------------------------
+:data:`pi`               *π* = 3.141592...
+:func:`e`                *e* = 2.718281...
+:func:`tau`              *τ* = 2\ *π* = 6.283185....
+:func:`inf`              Positive infinity.
+:func:`nan`              "Not a number" (NaN).
```

Comment:
These are data, not func — func adds parens!

Reply:
Indeed, I forgot to fix after the previous line. Thanks for noticing, I fixed it.